internal/testutil/orchestratortest/orchestrator.go

```diff
@@ -132,6 +132,12 @@ func NewAnotherCloudService() *orchestrator.CloudService {
 	}
 }
 
+var (
```

Comment:
can be joined with the `var` block below

Reply:
MockAssessmentResultRequest1 is of type orchestrator.GetAssessmentResultRequest, whereby MockAssessmentResult1 is of type assessment.AssessmentResult.

Should variables of different types be joined in Go?

Reply:
That’s not a problem